We do not need to autodetect SSL/UUID; SSL we do not support, UUID we always do.

--- a/Makefile.linux
+++ b/Makefile.linux
@@ -153,14 +153,18 @@ LDLIBS += $(shell $(PKG_CONFIG) --static
 LDLIBS += $(shell $(PKG_CONFIG) --static --libs-only-l libnetfilter_conntrack)
 endif # ($(TEST),1)
 
+ifeq ($(TARGET_OPENWRT),)
+# n/a - we don't enable https server for IGD v2 anyway in OpenWrt
 LDLIBS += $(shell $(PKG_CONFIG) --static --libs-only-l libssl)
 
+# n/a - we hardcodedly support libuuid
 TEST := $(shell $(PKG_CONFIG) --exists uuid && echo 1)
 ifeq ($(TEST),1)
 LDLIBS += $(shell $(PKG_CONFIG) --static --libs-only-l uuid)
 else
 $(info please install uuid-dev package / libuuid)
 endif # ($(TEST),1)
+endif
 
 TESTUPNPDESCGENOBJS = testupnpdescgen.o upnpdescgen.o
 
